index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="favicon.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <base href="/" />
```

Comment:
This should not be necessary

Reply:
If I didn't do this and go to the doctor's profile page, then I see no image from the doctor